docs/readme.html

```diff
@@ -243,8 +245,6 @@
   pc.setLocalDescription(myOffer);
   ```
 
-  _setLocalDescription_ indirectly controls the candidate gathering process.
```

Comment:
ok I'll test it again, in my tests it didn't happen before `setLocalDescription` but perhaps I was missing something. thanks!